compiler/lookups.nim

```diff
@@ -48,6 +48,11 @@ proc considerQuotedIdent*(c: PContext; n: PNode, origin: PNode = nil): PIdent =
         case x.kind
         of nkIdent: id.add(x.ident.s)
         of nkSym: id.add(x.sym.name.s)
+        of nkOpenSymChoice:
+          if x[0].kind == nkSym:
+            id.add(x[0].sym.name.s)
+          else:
+            handleError(n, origin)
```

Comment:
```suggestion
        of nkSymChoices: id.add(x[0].sym.name.s)
```

Alternatively we can also check `if x.len != 0` and error otherwise but this should ideally never be possible.